docs/key-features/features.md

```diff
@@ -102,6 +102,8 @@ karmada supports:
   * With ServiceExport and ServiceImport, achieving cross-cluster service discovery.
 * [Multi-cluster network support](https://karmada.io/docs/userguide/network/working-with-submariner):
   * Use `Submariner` to open up the container network between clusters.
+* [Cross-Cluster service discovery and failover via ErieCanal](https://karmada.io/docs/userguide/service/working-with-eriecanal)
```

Comment:
The link is broken?

Reply:
Bcoz the new doc exists in "current" version only, and the link points the latest release version (1.5).